Alleny244 patch 1

Alleny244 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes
### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
Yes

Alleny244 committed:
Added java program to implement sparse matrix

Alleny244 committed:
Updated Readme

Alleny244 committed:
Update README.md

HarshCasper commented on Java/ds/README.md:
```diff
@@ -1 +1,2 @@
-a
+
+*  [Sparse Matrix](SparseMatrix.java)
```
Don't make the README updates here

Alleny244 replied:
I have deletd that and updated the correct readme.md file

HarshCasper commented on Java/ds/SparseMatrices.java:
```diff
@@ -0,0 +1,120 @@
+            }
+            // First row's third column denotes
+            // number of non - zero elements in
+            // sparseMatrix
```
Use Multi-Line Comments

Alleny244 replied:
I have changed it into Multi-Line Comments

HarshCasper commented on Java/ds/SparseMatrices.java:
```diff
@@ -0,0 +1,120 @@
+//          SPARSE MATRIX
+
+
+//  Java program for Sparse Matrix Representation using Array 
```
Add more description

Alleny244 replied:
I have added more description

HarshCasper commented on Java/ds/SparseMatrices.java:
```diff
@@ -0,0 +1,120 @@
+
+public class SparseMatrices{
+
+    public static void main(String[] args) {
```
Use main() function only for the driver code. Use a separate function for putting all your logic

Alleny244 replied:
Changed as per the requirments

Alleny244 committed:
Upadated the code

Used Multi-Line Comments
Added more description
Used main() function only for the driver code and created a separate function for putting all the logic

Alleny244 committed:
Updated readme

Undoing the changes made to readme

Alleny244 committed:
Updated the correct readme

Alleny244 commented:
I have made the pull request after making all the changes

Alleny244 commented:
@HarshCasper can you review it again

ankitaggarwal23 reviewed (commented):
Problem already present `- [Sparse Matrix](ds/SparseMatrix.java)`
